File: 백준/Bronze/5596. 시험 점수/시험 점수.c
#include <stdio.h>

int main(void) { 
   int a, b, c, d;
   int sum1 = 0, sum2 = 0;
   scanf("%d %d %d %d", &a, &b, &c, &d);
   sum1 += a + b + c + d;

   scanf("%d %d %d %d", &a, &b, &c, &d);
   sum2 += a + b + c + d;

   if(sum1 < sum2){
      printf("%d", sum2);
   }
   else{
      printf("%d", sum1);
   }
   

   return 0;
}

File: 백준/Bronze/5596. 시험 점수/시험 점수.py
a, b, c, d = map(int, input().split())
sum1 = 0
sum1 += a + b + c + d

w, x, y, z = map(int, input().split())
sum2 = 0
sum2 += w + x + y + z

if sum1 < sum2:
  print(sum2)
else:
  print(sum1)